src/content/reference/cli.mdx

```diff
@@ -347,7 +347,7 @@ $ okteto init
 
 ### kubeconfig
 
-Download credentials for the Kubernetes cluster selected via `okteto context`.
+Download credentials for the Kubernetes cluster selected via `okteto context`. This credentials will internally use the `kubetoken` command below.
```

Comment:
@AgustinRamiroDiaz why do we need to share this with the user? Is this something they need to be aware of?